### ISSUE TITLE
scriptable command-line flags for `ionic start --cordova` that skips free account

### ISSUE THREAD

**Issue by reviewher** [ionic-team/ionic-cli#4769]
**Description:**

There is no set of command line flags for a non-interactive start of a new cordova app without the free account prompt.

When setting up a new project with `ionic start --cordova`, the first prompt is

```
[WARN] About to integrate your app with Cordova.
       
       We now recommend ⚡️ Capacitor⚡️  (https://ion.link/capacitor) as the official native runtime for Ionic. To
       learn about the differences between Capacitor and Cordova, see these docs[1]. For a getting started guide, see
       these docs[2].
       
       [1]: https://ion.link/capacitor-differences-with-cordova-docs
       [2]: https://ion.link/capacitor-using-with-ionic-docs
       
? Are you sure you want to continue? (Y/n) 
```

For this prompt, the answer is "Y".  Choosing "n" will fall back to Capacitor.  `--no-interactive` will select `n` here, so that option is not viable.  `--confirm` does not select "Y" so that option is not viable either.

The second prompt is

```
? Create free Ionic account? (y/N)
```

Here the desired choice is "N".

So in summary, the first question needs to be answered with "Y" and the second question has to be answered with "N", and the current set of command line flags do not cover this use case.

**Comment by BigBallard** [comment]
I second this, is there a way to flag this interaction?
